DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -141,8 +144,11 @@ const char* const TaggingVariableTokens[] = {
   /* [jetPt]                                    = */ "jetPt",
```

Comment:
@slava77 What do you mean? The descriptions are in the right handside in the strings.